torchmetrics/functional/text/mer.py

```diff
@@ -63,26 +64,50 @@ def _mer_compute(errors: Tensor, total: Tensor) -> Tensor:
 
 
 def match_error_rate(
-    predictions: Union[str, List[str]],
-    references: Union[str, List[str]],
+    preds: Union[None, str, List[str]] = None,
+    target: Union[None, str, List[str]] = None,
+    predictions: Union[None, str, List[str]] = None,
+    references: Union[None, str, List[str]] = None,
 ) -> Tensor:
     """Match error rate is a metric of the performance of an automatic speech recognition system. This value
     indicates the percentage of words that were incorrectly predicted and inserted. The lower the value, the better
     the performance of the ASR system with a MatchErrorRate of 0 being a perfect score.
 
     Args:
-        predictions: Transcription(s) to score as a string or list of strings
-        references: Reference(s) for each speech input as a string or list of strings
+        preds: Transcription(s) to score as a string or list of strings
+        target: Reference(s) for each speech input as a string or list of strings
 
     Returns:
         Match error rate score
 
     Examples:
-        >>> predictions = ["this is the prediction", "there is an other sample"]
-        >>> references = ["this is the reference", "there is another one"]
-        >>> match_error_rate(predictions=predictions, references=references)
+        >>> preds = ["this is the prediction", "there is an other sample"]
+        >>> target = ["this is the reference", "there is another one"]
+        >>> match_error_rate(preds=preds, target=target)
         tensor(0.4444)
     """
-
-    errors, total = _mer_update(predictions, references)
+    if preds is None and predictions is None:
+        raise ValueError("Either `preds` or `predictions` must be provided.")
+    if target is None and references is None:
+        raise ValueError("Either `target` or `references` must be provided.")
```

Comment:
thinking that we shall also raise an error in case that the duplicated args are set
```py
    if preds is not None and predictions is not None:
        raise ValueError("Only one from `preds` or `predictions` must be provided.")
    if target is not None and references is not None:
        raise ValueError("Only one from `target` or `references` must be provided.")
```

Reply:
This case seems unlikely.